examples/apps/presence-tracker/tsconfig.json

```diff
@@ -4,6 +4,7 @@
 		"outDir": "./lib",
 		"types": ["jest", "puppeteer", "jest-environment-puppeteer", "expect-puppeteer"],
 		"exactOptionalPropertyTypes": false,
+		"noUnusedLocals": false,
```

Comment:
??

Reply:
Debugging. I really wish we checked for unused variables in the lint phase instead of compile.